contrib/ceph-versioned-image-descriptions.md

```diff
@@ -0,0 +1,55 @@
+DockerHub descriptions for ceph-vX.Y.Z images
+==============================================
+
+ceph/ceph
+----------
+
+### Short Description
+Images containing all Ceph binaries.
+
+### Full Description
+Images contain all Ceph binaries as well as NFS-Ganesha and iSCSI binaries since these components
+are heavily tied to Ceph's version to be compatible. Unless otherwise noted in the tag, all images
+are CentOS based. New images are built within 24 hours of a new Ceph version being published to
+the official Ceph package repository.
+
+#### Image tag breakdown
+
+**Build date** </br>
+Some images are suffixed with an 8-digit build date in the form `YYYMMDD`. This indicates the date
+the image was built and is used similarly to a build number for typical packages. For an image which
+already exists (e.g., `v12.2.7-20181023`), a new version with a different build date will be built
+when there is a base image update. For example, if the CentOS base image gets a security fix on
+10 February 2080, the example image above will get a new image built with tag `v12.2.7-20800210`.
+
+**Versions** </br>
+There are a few ways to choose the Ceph version you desire:
+ - Full semantic version with build date, e.g., `v12.2.9-20181026`
+   - These tags are intended for use when precise control over image upgrade is desired and are
+     recommended for production use.
+ - Major version, e.g. `v12` (a.k.a. _Ceph Luminous_)
+   - These tags are always the most recent build of the newest Ceph major release matching the tag.
+ - Minor version (e.g., `v12.1`)
```

Comment:
seems like we don't need the `v12.1` or `v12.2` tags. With ceph's versioning scheme, all the stable releases have the minor version of `.2`. Thus, `v12` would already cover the latest stable release and would be redundant with the tag `v12.2`.

Reply:
I agree that they don't seem terribly useful. But they exist and so are documented here.